Library/Homebrew/utils.rb

```diff
@@ -84,7 +84,7 @@ def inject_dump_stats!(the_module, pattern)
   # rubocop:enable Style/GlobalVars
 end
 
-module Kernel
+module Kernel # rubocop:disable Style/Documentation
```

Comment:
```
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/utils.rb:87:15: W: [Correctable] Lint/RedundantCopDisableDirective: Unnecessary disabling of Style/Documentation.
[11](https://github.com/Homebrew/brew/actions/runs/4275555139/jobs/7442995057#step:6:12)
module Kernel # rubocop:disable Style/Documentation
[12](https://github.com/Homebrew/brew/actions/runs/4275555139/jobs/7442995057#step:6:13)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

❓

Reply:
I think this is the "invoking `brew style` from `/opt/homebrew` or `/opt/homebrew/Library/Homebrew` issue again.

Reply:
The easiest way to fix this is to add a comment to `module Kernel` in `Homebrew/utils.rb` and then the test passes and the linter is happy. No more in-line excludes.